### PR TITLE
Made language a little clearer.

### PR DESCRIPTION
**Pull request by sheeldotme**
"This" in the top comment is a little unclear as it's unclear whether that comment refers to just the next message type or the entire block.
"Those" on the bottom comment is a little unclear as "those" is past tense, it could refer to the preceding message types.

TODO:

- [x] Make sure all of the significant new logic is covered by tests
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests and linter rules pass
- [x] Update CHANGELOG.md with your change

### CODE REVIEW

File: src/message-types.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ export default class MessageTypes {
   public static GQL_CONNECTION_ACK = 'connection_ack'; // Server -> Client
   public static GQL_CONNECTION_ERROR = 'connection_error'; // Server -> Client
 
-  // NOTE: This one here don't follow the standard due to connection optimization
+  // NOTE: These message types do not follow the standard due to connection optimizations
```
Comment (DxCx) [inline src/message-types.ts]: this is not true for all message types, just for KEEP_ALIVE..

File: src/message-types.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ export default class MessageTypes {
   public static GQL_COMPLETE = 'complete'; // Server -> Client
   public static GQL_STOP = 'stop'; // Client -> Server
 
-  // NOTE: Those message types are deprecated and will be removed soon.
+  // NOTE: These message types are deprecated and will be removed soon.
```
Comment (DxCx) [inline src/message-types.ts]: 👍

**Comment by stubailo** [comment]
@sheeldotme any updates on this?

**Comment by sheeldotme** [comment]
I think the fact that I tried to interpret it the best I could, and failed shows there's some opportunity for clarification here.  I've taken @DxCx's helpful feedback into account and updated the comment.

**Comment by NeoPhi** [comment]
Thank you for the documentation update!

**Pull request merged**